frontend/packages/dev-console/src/components/edit-application/EditApplicationComponent.tsx

```diff
@@ -26,13 +26,24 @@ const EditApplicationComponent: React.FunctionComponent<EditApplicationComponent
   const { appName, resources } = props;
   const appLabel = resources.editAppResource?.data?.metadata?.labels?.['app.kubernetes.io/name'];
```

Comment:
```suggestion
  const appLabel = resources.editAppResource?.data?.metadata?.labels?.['app.kubernetes.io/name'];
```

No fallback to the old INSTANCE_LABEL here?